src/main/java/mate/academy/mapper/BookMapper.java

```diff
@@ -0,0 +1,14 @@
+package mate.academy.mapper;
+
+import mate.academy.config.MapperConfig;
+import mate.academy.dto.BookDto;
+import mate.academy.dto.CreateBookRequestDto;
+import mate.academy.model.Book;
+import org.mapstruct.Mapper;
+
+@Mapper(config = MapperConfig.class)
+public interface BookMapper {
+    BookDto toDto(Book book);
```

Comment:
Consider adding null checks or handling for the 'book' parameter in the 'toDto' method to avoid potential NullPointerExceptions.